manifest/dev.baru/NeosBetterIMESupport/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "NeosBetterIMESupport",
```

Comment:
Probably will want to rename at some point to avoid confusion. Can do so when next updating